packages/tasit-action/src/contract/Contract.test.js

```diff
@@ -147,7 +147,7 @@ describe("TasitAction.Contract", () => {
       });
 
       it("and Action error event on action error", async () => {
-        const contractErrorListener = sinon.fake(() => {
+        const contractErrorListener = sinon.fake(error => {
           sampleContract.off("error");
```

Comment:
We probably don't actually need to remove the error listener in a test like this, right? Is it here because of the instantaneous block times?

Reply:
Right. The `Contract.test.js` differently from `Decentraland.test.js` is cleaning listener between test cases. That will be addressed on a PR that I'm working on.

Reply:
Okay, great. Thanks for the info and update.

Reply:
Could I close this?

Reply:
Yep. Thanks!